### PR TITLE
Add alerts APIs

### PR DESCRIPTION
**Pull request by hfm**
reference:

* http://help.mackerel.io/entry/spec/api/v0#alert-get
* http://help.mackerel.io/entry/spec/api/v0#alert-close

### CODE REVIEW

**Comment by udzura** [comment]
:+1: to me

File: spec/client/alert_spec.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+describe Mackerel::Client::Alert do
+  let(:alert_id) { 'DUMMY' }
+
+  describe '.alerts' do
+    it 'returns all alerts' do
+      stub_api(:get, '/v0/alerts')
+      res = Mackerel.alerts
+      expect(res).to be_a Array
```
Comment (hsbt) [inline spec/client/alert_spec.rb]: Please add assertion of body inspection like `expect(res.size).to be 1`.

Comment (hfm) [inline spec/client/alert_spec.rb]: Added spec by c1fb6b7 :smile:

**Comment by hsbt** [comment]
:octocat:

**Pull request merged**